### PR TITLE
Enable tilt

### PR DESCRIPTION
**Pull request by evan-gray**

### CODE REVIEW

File: .github/workflows/tilt.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - uses: actions/checkout@v2
+      - name: Expand for link to Tilt dashboard (only available during build)
+        run: >
+          echo "Tilt progress dashboard: https://$DASHBOARD_URL"
+      - run: |
+          kubectl config set-context ci --namespace=$DEPLOY_NS
+          kubectl config use-context ci
+
+      - run: tilt ci -- --evm2 --namespace=$DEPLOY_NS
+        timeout-minutes: 60
```
Comment (djb15) [inline .github/workflows/tilt.yml:36]: The default timeout for `tilt ci` is 30 mins btw despite having the longer timeout on the step. Don't think we should need to increase beyond 30 though

**Review by djb15 — APPROVED** [review]
Glossed over the bindings but I'm on-board with enabling tilt tests

File: .github/workflows/tilt.yml (hunk 1 above)
Comment (evan-gray) [reply .github/workflows/tilt.yml:36]: agree, this was copied from the wormhole repo. would rather have `tilt ci` timeout than GH kill the step, I guess :shrug: open to changing in a future PR

**Comment by johnsaigle** [comment]
Looks good to me! I am not a reviewer on the repo though so you'll still need someone else to weigh in 😄 

My only note is that GitHub is flagging the branch as out-of-date. Might be worth a rebase-and-push in case GitHub doesn't allow you to merge because of this?

**Review by johnsaigle — APPROVED** [review]

**Comment by evan-gray** [comment]
will follow up with another regen in #229 which also contains some test fixes

**Pull request merged**